CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -581,6 +580,9 @@ local function setupGamepadControls()
 			gamepadSettingsFrame:TweenSizeAndPosition(UDim2.new(0,102,0,102), UDim2.new(0.5,-51,0.5,-51),
 														Enum.EasingDirection.Out, Enum.EasingStyle.Sine, 0.1, true, 
 				function()
+					if not InputService.VREnabled then
+						pcall(function() InputService.OverrideMouseIconBehavior = Enum.OverrideMouseIconBehavior.None end)
```

Comment:
What I mean is we should improve the `setOverrideMouseIconBehavior` function and then invoke that function instead of just setting the mouse behavior here with no regard to the function.